src/components/ChatRoomPage/Chat.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IChat } from '../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { currentUserState } from '../../state/atom';
+import userData from '../../db/userData.json';
+
+const Chat = (chat: IChat) => {
+  const currentUser = useRecoilValue(currentUserState);
+  //chat.userId === currentUser.userId;
+
+  return (
+    <ChatContainer isActive={chat.userId === currentUser.userId}>
+      <UserImg
+        src={`${process.env.PUBLIC_URL}/Imgs/${
+          userData[chat.userId].userImg
+        }.jpg`}
+        alt={userData[chat.userId].userName}
+        isActive={chat.userId === currentUser.userId}
+      />
+      <div>
+        <UserName isActive={chat.userId === currentUser.userId}>
+          {userData[chat.userId].userName}
+        </UserName>
+        <ChatBox isActive={chat.userId === currentUser.userId}>
+          <ChatContent isActive={chat.userId === currentUser.userId}>
+            {chat.content}
+          </ChatContent>
+          <ChatTime>{chat.time}</ChatTime>
+        </ChatBox>
+      </div>
+    </ChatContainer>
+  );
+};
+
+export default Chat;
+
+const ChatContainer = styled.div<{ isActive: boolean }>`
+  display: flex;
+  justify-content: flex-start;
+  padding: 0.6rem 0.7rem;
+
+  flex-direction: ${(props) => (props.isActive ? 'row-reverse' : 'row')};
+`;
+
+const UserImg = styled.img<{ isActive: boolean }>`
+  width: 3.5rem;
+  height: 3.5rem;
+  border-radius: 50%;
```

Comment:
```suggestion
  width: 3.5rem;
  height: 3.5rem;
  border-radius: 50%;
  background-image: url(${(props) => (props.isActive ? user1 : user2)});
```
isActive 를 props로 받으신 걸 보니... isActive 상태에 따라 이미지를 바꾸고 싶으셨던 걸까용???
그렇다면, 요런식으로 수정이 가능합니다!!
user1, user2는 각각 해당하는 경로에서 import 해주는 작업이 추가로 필요하긴 합니다!